l10n_es_account_bank_statement_import_n43/wizards/account_bank_statement_import_n43.py

```diff
@@ -169,6 +174,8 @@ def _parse(self, data_file):
         for raw_line in data_file.split("\n"):
             if not raw_line.strip():
                 continue
+            if raw_line in ignore_lines:
```

Comment:
Mejor hacer la comparación arriba:

`if not raw_line.strip() or raw_line in ignore_lines:`